README.md

```diff
@@ -101,25 +95,24 @@ precision while being roughly twice as fast and requiring half the amount of GPU
 import torch
 from diffusers import StableDiffusionPipeline
 
-pipe = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16, revision="fp16")
+pipe = StableDiffusionPipeline.from_pretrained("runwayml/stable-diffusion-v1-5", torch_dtype=torch.float16)
```

Comment:
Why remove the revision here?

Reply:
Good question! In short, I'd like to move away from using revisions because:
a) It seems like revisions are not good practice to maintain a variation of the model
b) branches on the Hub cannot be contributed to or adapted by the community
c) No community member has every used a revision in diffusers so far
d) it's not needed to have the model run in fp16 - it's only needed to save some bandwith when loading the model and initial CPU memory 
e) It's confusing for the community as they might think all checkpoints have this option but they don't